server/routes/api/user.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config();
+const express = require('express');
+const db = require('../../db');
+
+const router = express.Router();
+
+// It's the server responsibility to identify the client,
+// for now the client will be identified by the '.env'
+const CLIENT_UPN = process.env.CLIENT_UPN || 's8182384';
```

Comment:
I dislike that `s8182384 is a magic personal id`

Please do something like: `const CANDIDATE_UPN = 's8182384'`. Same goes about `const EMPLOYER_UPN = '<UPN_OF_RAMAD>'`

Reply:
I dont like it because youre hacking the current authorization provider.  
IMO, we should use the id given by our current provider (github) as the UPN field.
In regards to permissions (if the user is ramad or not) i added a table holding the ramads for each department.
For now as we are not developing in our rnvironment we should'nt make patches if there is a more suitable solutio n and I belive that using the github username as the upn is more elegant and fits here like a glove ✋

Reply:
and for dev environment I use another env var to indicates if a permission check is needed

Reply:
@SteveVaknin  Could you explain how? I simply turn off the authentication locally... and not push the turn off

Reply:
סבבה נדבר על ה מחר

Reply:
כבר לא בשימוש